public/views/index-template.html

```diff
@@ -199,9 +199,10 @@
         </p>
         <p>
           1. This could be caused by your reverse proxy settings.<br /><br />
-          2. If you host grafana under subpath make sure your grafana.ini root_url setting includes subpath<br />
+          2. Otherwise, if you host grafana under subpath make sure your grafana.ini root_url setting includes subpath
+          and that serve_from_sub_path is set.<br />
```

Comment:
serve_from_sub_path shouldn't be needed when using a reverse proxy, but could. You use serve_from_sub_path if you want to host under a sub path but don't use a reverse proxy.

Reply:
that's I have used `otherwise`. Should I change it to: `I you don't use a reverse proxy...`?

Reply:
I'm confused by the whole issue request. If you're not using a reverse proxy why would you end up with this problem?

Reply:
I guess the [issue](https://github.com/grafana/grafana/issues/26128) is for adding an indication for making sure that `serve_from_sub_path` is set in addition to setting `root_url` properly  if not using a reverse proxy, no?